tesseract_time_parameterization/test/time_optimal_trajectory_generation_tests.cpp

```diff
@@ -228,6 +229,78 @@ TEST(time_optimal_trajectory_generation, testLargeAccel)
   }
 }
 
+// Caues the issue below. Workaround is applied when using computeTimestamps interface like in
+// testCommandLanguageInterface https://github.com/ros-industrial-consortium/tesseract_planning/issues/27
+// TEST(time_optimal_trajectory_generation, test_return_home)
+//{
+//  Eigen::VectorXd waypoint(7);
+//  std::list<Eigen::VectorXd> waypoints;
+
+//  waypoint << 0, 0.7, -2.1, 0, -0.25, 0;
+//  waypoints.push_back(waypoint);
+//  waypoint << 0, 0, 0, 0, 0, 0;
+//  waypoints.push_back(waypoint);
+//  waypoint << 0, 0.70001, -2.1, 0, -0.25, 0;
+//  waypoints.push_back(waypoint);
+//  waypoint << 0, 0, 0, 0, 0, 0.1;
+//  waypoints.push_back(waypoint);
+
+//  Eigen::VectorXd max_velocities(7);
+//  max_velocities.setOnes();
+//  Eigen::VectorXd max_accelerations(7);
+//  max_accelerations.setOnes();
+
+//  Trajectory trajectory(Path(waypoints, 0.001), max_velocities, max_accelerations, 0.001);
+//  EXPECT_TRUE(trajectory.isValid());
+//}
```

Comment:
It looks likes this was used to test the approach you applied; should it be removed?

Reply:
This test still fails because of the bug in trajectory. The workaround only is applied when using the computeTimesteps function. I can remove it if you want. I left it in there so we could test if any of the changes you were making fixed the underlying issue.

Reply:
> Ah, so this is because it visits the same location twice in the same trajectory?

I think it is just if it revisits the start or end location.

Reply:
> This test still fails because of the bug in trajectory.

Is this bug captured somewhere; I just do not want to loose track of it? It may also be good to leverage the doxygen "@bug" functionality so it also shows up in the documentation. 

>  I left it in there so we could test if any of the changes you were making fixed the underlying issue.

I am good with it staying.